docs/troubleshooting/windows-pv-tools.md

```diff
@@ -80,6 +69,76 @@ Consult your motherboard manual for details; for example, on Dell systems with i
 ![](../../static/img/performance-setting.png)
 </div>
 
+## Completely removing existing Xen PV drivers with XenClean
+
+XenClean is an utility for cleanly removing Xen PV drivers and management agents of the following products:
+
+* XCP-ng Windows PV Tools, versions 8.2 and 9.1
+* XenServer VM Tools for Windows, versions 9.3 and 9.4
+* Other Xen drivers
+
+It is included in the installation package of XCP-ng Windows PV Tools 9.1 and above.
+[See the newest releases here.](https://github.com/xcp-ng/win-pv-drivers/releases)
+
+To use XenClean, simply run the `Invoke-XenClean.ps1` script **as Administrator**. Your system will automatically reboot.
```

Comment:
Maybe we already advise to snapshot the VM elsewhere, but I'd remind users here before this step.